content/applications/finance/fiscal_localizations/belgium.rst

```diff
@@ -0,0 +1,164 @@
+
+.. seealso::
+   More information concerning the Fiscal Data Module can be found on `the official website
+   <https://www.systemedecaisseenregistreuse.be/>`_.
+
+Configuration
+~~~~~~~~~~~~~
+
+Before setting up your database to work with a FDM, ensure you have the following hardware:
+
+- a registered :ref:`black box <belgium/blackbox>` (go to
+  `www.boîtenoire.be <https://www.xn--botenoire-r5a.be/>`_ to order yours and to the `FPS website
+  <https://finances.belgium.be/E-services/registratiemodule_gks/registratiemodule_gerigstreerd_kassasysteem>`_
+  to register it);
+- a serial null modem cable per FDM (`example
+  <https://www.startech.com/Cables/Serial-Parallel-PS-2/DB9-DB25/10-ft-Cross-Wired-Serial-Null-Modem-Cable-DB9-FM~SCNM9FM>`__);
+- a serial-to-USB adapter per FDM (`example
+  <https://trendnet.com/products/proddetail.asp?prod=265_TU-S9>`__); and
+- An :ref:`IoT Box <belgium/iotbox>` (one IoT box per black box).
+
+.. _belgium/blackbox:
+
+Black box
+*********
+
+As a pre-requisite, activate the **black box module**. To do so, go to the :guilabel:`Apps`
+application, remove the :guilabel:`Apps` filter, search for the ``Belgian Registered Cash
+Register`` module (technical name: ``pos_blackbox_be``) and click :guilabel:`Activate`.
+
+.. image:: belgium/be-modules.png
+   :align: center
+   :alt: black box modules for belgian fiscal certification
+
+Once the module is activated, update your company info with your VAT number. To set it up, go to
+:menuselection:`Settings --> Companies --> Update Info`, and fill in the :guilabel:`VAT` field.
+Then, enter a national registration number for every staff member who will be operating the POS
+system. To do so, go to the :guilabel:`Employees` app and open an employee form. After that, go to
+the :menuselection:`HR settings tab --> Attendance` and fill in the :guilabel:`INSZ or BIS number`
+field.
+
+.. image:: belgium/bis-number.png
+   :align: center
+   :alt: ISNZ or BIS number field on employee form
+
+.. tip::
+   To input your information, click on your avatar, go to :menuselection:`My Profile --> Preference
+   tab`, and enter your INSZ or BIS number in the designated field.
+
+When you open the Point of Sale and make your initial transaction, you will be prompted to enter the
+PIN that was provided with your :ref:`VAT signing card <belgium/introduction>`.
+
+.. image:: belgium/vsc-pin.png
+   :align: center
+   :alt: VAT Signing Card PIN initialization
+
+.. warning::
+   It is essential to configure the black box directly in the production database, as utilizing it
+   in a testing environment may result in incorrect data being stored within the black box.
+
+.. _belgium/iotbox:
+
+IoT Box
+*******
+
+In order to use a Fiscal Data Module, you need a registered IoT Box. To register your IoT box, you
+must contact us through our `contact form <https://www.odoo.com/help>`_, and provide the following
+information:
+
```

Comment:
I don't really know the info they have once they contact the help service, but they have to register as well on the SPF.
It can be done here: https://eservices.minfin.fgov.be/gks/jsf/external/horeca

Reply:
Wouldn't they (the sales person and the customer) be doing the same thing? Or do they both have to register?